plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -478,15 +478,15 @@ def check_ber_issue(self, port_obj, row, timestamp):
                         return Issue(port_obj.port_name, Constants.ISSUE_BER)
             return None
 
-        def read_next_set_of_high_ber_or_pdr_ports(self):
+        def read_next_set_of_high_ber_or_pdr_ports(self, ports_data, ports_counters):
```

Comment:
lets rename to something like "analyze_telemetry_data"

Reply:
Ok